pkg/controller/amazoncloudintegration/amazoncloudintegration_controller.go

```diff
@@ -248,9 +247,8 @@ func (r *ReconcileAmazonCloudIntegration) Reconcile(ctx context.Context, request
 	r.status.ClearDegraded()
 
 	if !r.status.IsAvailable() {
-		// Schedule a kick to check again in the near future. Hopefully by then
-		// things will be available.
-		return reconcile.Result{RequeueAfter: 30 * time.Second}, nil
+		// Schedule a kick to check again in the near future. Hopefully by then things will be available.
+		return reconcile.Result{RequeueAfter: utils.StandardRetry}, nil
```

Comment:
I've replaced a bunch of places with a "StandardRetry" on the premise that it doesn't actually matter if it's 5 seconds, 10 seconds, 15 seconds, or 30 seconds. We were using all of those (and maybe more) in various places, and magic numbers are "Bad".  

Let me know if that assumption is wrong and we do actually need second level precision for some of these retries. If so, I can introduce a second or third constant with an appropriate name that explains why a faster or slower retry is actually needed for this particular case.